lib/server/cache_handler.es6.js

```diff
@@ -0,0 +1,272 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'ClockCache',
```

Comment:
Add documentation. Is this approximately an LRU cache?

Reply:
It is [exactly that](https://en.wikipedia.org/wiki/Page_replacement_algorithm#Clock).

Reply:
... willl add documentation.